frontend/src/components/navbar/Search.jsx

```diff
@@ -1,17 +1,55 @@
-import React, { useState } from "react";
+import React from "react";
 
 function Search() {
-  const [city, setCity] = useState("");
   return (
-    <div className="nav-search">
-      <input
-        value={city}
-        className="search"
-        type="text"
-        placeholder="Recherche"
-        onChange={(event) => setCity(event.target.value)}
-      />
-    </div>
+    <form className="bg-gray-100 space-y-10 flex justify-center items-center">
+      <div className="flex items-center p-1 space-x-6 bg-[#3589a7] ">
```

Comment:
This div seems optionnal because your form already round your content. If you use it for style, you might apply the style to your form.